pkgs/top-level/all-packages.nix

```
@@ -13621,6 +13621,8 @@ in
 
   mailman = callPackage ../servers/mail/mailman { };
 
+  mailman-rss = python3Packages.callPackage ../development/python-modules/mailman-rss { };
```

```suggestion
  mailman-rss = callPackage ../development/python-modules/mailman-rss { };
```
We prefer having `python3(Packages)` as argument so that Python packages can be overridden per application.